README.md

```diff
@@ -1 +1,42 @@
-# geckos-14
\ No newline at end of file
+# DashThings.io
+
+DashThing.io is a tool dedicated to DIYers in need of an easy way to build
+**Dashboard** for their **Things**.
+
+## Context
+
+We are using the concept of "Things" described in the "Building the Web of
+Things" book writen by Dominique D. Guinard and Vlad M. Trifa:
+
+_A Thing with an uppercase "T" refers to a smart thing. A physical object that's
+digitally augmented with one or more of the following:_
```

Comment:
i'm not sure what you mean by digitally augmented- say if it's sensor it is gonna be physically attached to the board?